### PR TITLE
PR for BXMSDOC-7312: Release notes for 7.11 DM and PAM

### PR DESCRIPTION
**Pull request by kaldesai**
- Epic: https://issues.redhat.com/browse/BXMSDOC-7312
- RHPAM: http://file.pnq.redhat.com/~kaldesai/BXMSDOC-7312-RN-7.11-RHPAM-Final/
- RHDM: http://file.pnq.redhat.com/~kaldesai/BXMSDOC-7312-RN-7.11-RHDM-Final/

### CODE REVIEW

File: doc-content/enterprise-only/release-notes/rn-kogito-productize-features-ref.adoc [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
-* https://marketplace.visualstudio.com/items?itemName=redhat.vscode-extension-red-hat-business-automation-bundle[_Red Hat Business Automation Bundle_]
+{PRODUCT} now supports {KOGITO} JVM Quarkus runtime images.
+
+== Support for {KOGITO} builder images for Qarkus and SpringBoot
+
+{PRODUCT} now includes {KOGITO} builder images for Quarkus and SpringBoot.
+
+ifdef::PAM[]
+
+== Support for {KOGITO} operator on {OPENSHIFT} 4
```
Comment (krisv) [inline doc-content/enterprise-only/release-notes/rn-kogito-productize-features-ref.adoc:22]: This title does not seem to match description?  The description doesn't have anything to do with Kogito, but is about integrating RHPAM applications (on Kie-server) with Kafka, that is now also available when deploying on OpenShift (but this already included lower in section 3.9.1), so seems description needs to be updated here to reflect the title?

File: doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc [hunk 2]
```diff
@@ -5,191 +5,202 @@ This section highlights new features in {PRODUCT} {PRODUCT_VERSION}.
...
+You can now use the `RecordsPerTransaction` parameter with the `LogCleanupCommand` executor command that indicates the number of records in a transaction that is removed. For more information about `RecordsPerTransaction` parameter, see {URL_DEVELOPING_PROCESS_SERVICES}#assembly-designing-business-processes[_{DESIGNING_BUSINESS_PROCESSES}_].
 
-{PRODUCT} {ENTERPRISE_VERSION} includes a new *Decision Service details* section in the DMN editor properties panel containing a read-only list of *Input Data*, *Encapsulated Decisions*, and *Output Decisions*.
+=== Standalone BPMN editor now supports readonly mode
 
-For more information, see {URL_GETTING_STARTED}#assembly-getting-started-decision-services[_{GETTING_STARTED_DECISION_SERVICE}_] and {URL_DEVELOPING_DECISION_SERVICES}#assembly-dmn-models[_{DMN_MODELS}_].
+You can now use the standalone BPMN editor in readonly mode. For more information about standalone editors, see {URL_DEVELOPING_PROCESS_SERVICES}#assembly-designing-business-processes[_{DESIGNING_BUSINESS_PROCESSES}_].
 
-ifdef::PAM[]
+=== Support for work item definitions in VSCode extension and standalone editors
```
Comment (krisv) [inline doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:32]: This is about VSCode and standalone, don't think it makes sense under the "business central" category.

File: doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc [hunk 3]
```diff
@@ -5,191 +5,202 @@ This section highlights new features in {PRODUCT} {PRODUCT_VERSION}.
...
+For more information, see {URL_DEVELOPING_DECISION_SERVICES}#assembly-test-scenarios[_{TESTING_DECISION_SERVICE}_].
 
-* Terminal
+ifdef::PAM[]
 
-. Launch VSCode Quick Open.
-. Enter the following command and press *Enter*.
-+
-`ext install redhat.vscode-extension-red-hat-business-automation-bundle`
+=== New `user` parameter in KIE REST API endpoint
```
Comment (krisv) [inline doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:77]: Think this should be moved under Kie-server

File: doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc [hunk 4]
```diff
@@ -5,191 +5,202 @@ This section highlights new features in {PRODUCT} {PRODUCT_VERSION}.
...
 
-For more information, see {URL_DEVELOPING_DECISION_SERVICES}#assembly-dmn-models[_{DMN_MODELS}_] and {URL_DEVELOPING_PROCESS_SERVICES}#assembly-designing-business-processes[_{DESIGNING_BUSINESS_PROCESSES}_].
+You can now use the new `user` parameter in a KIE REST API endpoint to bypass the enabled authenticated user. The `user` parameter is added for adding or removing the users or groups to a task.
 
+endif::[]
 
-ifdef::PAM[]
+=== PMML trusty implementation
```
Comment (krisv) [inline doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:83]: Think this should be moved under Kie-server

File: doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc [hunk 5]
```diff
@@ -5,191 +5,202 @@ This section highlights new features in {PRODUCT} {PRODUCT_VERSION}.
...
 
-== Process Designer
+You can now use PMML trusty implementation to embed a PMML call in a Java application or to execute a PMML model using {KIE_SERVER}. For more information about PMML trusty implementation, see {URL_DEVELOPING_DECISION_SERVICES}#assembly-pmml-models[_{PMML_MODELS}_].
 
-=== Ability to access activity details in boundary events
+=== REST endpoints for specific DMN models
```
Comment (krisv) [inline doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:87]: Think this should be moved under Kie-server

**Review by krisv — CHANGES_REQUESTED** [review]

File: doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc (hunk 2 above)
Comment (kaldesai) [reply doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:32]: Done

File: doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc (hunk 3 above)
Comment (kaldesai) [reply doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:77]: Updated

File: doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc (hunk 4 above)
Comment (kaldesai) [reply doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:83]: Updated

File: doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc (hunk 5 above)
Comment (kaldesai) [reply doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc:87]: Updated

**Pull request merged**